src/plugins/GUITestBase/src/GTUtilsTaskTreeView.h

```diff
@@ -75,6 +80,12 @@ class GTUtilsTaskTreeView {
 
     static int countTasks(HI::GUITestOpStatus& os, const QString& itemName);
     static QString getTaskStatus(HI::GUITestOpStatus& os, const QString& itemName);
+    /*f
```

Comment:
nit: 
1) remove 'f'
2) Rename 'findIfNotFound' to 'failIfNotFound'

Reply:
OK